cheatsheets/Transport_Layer_Security_Cheat_Sheet.md

```diff
@@ -36,17 +36,35 @@ There are a large number of different ciphers (or cipher suites) that are suppor
 
 The Mozilla Foundation provides an [easy-to-use secure configuration generator](https://ssl-config.mozilla.org/) for web, database and mail servers. This tool allows site administrators to select the software they are using and receive a configuration file that is optimized to balance security and compatibility for a wide variety of browser versions and server software.
 
-### Use Strong Diffie-Hellman Parameters
+### Do not explicitly set Diffie-Hellman Parameters
 
-Where ciphers that use the ephemeral Diffie-Hellman key exchange are in use (signified by the "DHE" or "EDH" strings in the cipher name) sufficiently secure Diffie-Hellman parameters (at least 2048 bits) should be used
+Whereas in the past the TLS protocol required to set Diffie-Hellman parameters for use by the ephemeral Diffie-Hellman key exchange (signified by the "DHE" or "EDH" strings in the cipher suite name), this caused practical issues as the client had no say in the selection of server parameters, meaning could only unconditionally accept or drop, and even resulted to denial of service attacks (CVE-2022-40735, CVE-2002-20001). 
```

Comment:
Word missing (presumably 'it' or perhaps 'the client') in the clause "meaning could only unconditionally accept or drop".